myapp/app/helpers/tasks_helper.rb

```diff
@@ -1,4 +1,13 @@
 # frozen_string_literal: true
 
 module TasksHelper
+  def sort_order(column, title)
+    (direction = column == sort_column && sort_direction == 'asc') ? 'desc' : 'asc'
```

Comment:
これって動きました？
見栄え的に代入は括弧の外の方がいいかもしれません！
```suggestion
    direction = (column == sort_column && sort_direction == 'asc') ? 'desc' : 'asc'
```

Reply:
かっこの位置をミスったのはケアレスミスだったんですが、条件をかっこで囲むとfablicopに怒られてしまいました。
Rubyとしてはかっこで囲わない流儀のようですね...
![image](https://user-images.githubusercontent.com/90371497/144158253-e27e0f50-8043-44cd-9816-2bf41ce3b769.png)

Reply:
あらw
そしたら行数増えちゃいますがこんなんでどうでしょう？
可読性は補えるかと！
```
direction = 'asc'
if  column == sort_column && sort_direction == 'asc'
  direction = 'desc'
end
```

Reply:
fablicopさんにこんな感じに直されましたw
```ruby
    direction = 'asc'
    direction = 'desc' if column == sort_column && sort_direction == 'asc'
```

Reply:
ぴえんw
でしたらそのままでOKでず！

Reply:
了解です